tinygrad/shape/symbolic.py

```diff
@@ -75,8 +76,12 @@ def __floordiv__(self, b:int, factoring_allowed=True):
       return (self + -offset*b).__floordiv__(b, factoring_allowed=False) + offset
     return create_node(DivNode(self, b))
 
-  def __mod__(self, b:int):
+  def __rmod__(self, b:int):
+    if self.min > b >= 0: return NumNode(b)
```

Comment:
Is there a test for this? Does the fuzzer hit this?

Reply:
it was implicitly tested in one of the test, and I added a direct test case against it. fuzzer does not hit this because in fuzzer all LHS is a `Node`. For general fuzzer support i will reply in another comment